test/system/700-play.bats

```diff
@@ -1090,6 +1090,87 @@ EOF
     run_podman rmi $imgname1 $imgname2
 }
 
+@test "podman play with image volumes" {
```

Comment:
This seems to be copy pasta from the previous test, please just merge them.
Building the same images several times make the tests slow.

Reply:
Makes sense

Reply:
Test updated